app/controllers/katello/api/v2/flatpak_remote_repositories_controller.rb

```diff
@@ -53,6 +53,8 @@ def show
     api :POST, "/flatpak_remote_repositories/:id/mirror", N_("Mirror a flatpak remote repository")
     param :id, :number, :desc => N_("Flatpak remote repository numeric identifier"), :required => true
     param :product_id, :number, :desc => N_("Product ID to mirror the remote repository to"), :required => true
```

Comment:
```suggestion
    param :product_id, :number, :desc => N_("Product ID to mirror the remote repository to")
```
No longer required param.
However we do want to validate that one of the product_id or Product name + org is passed..

Reply:
Something like we do here:
https://github.com/Katello/katello/blob/14e1906cda0b473075f395ec67ab1e107cc2b85c/app/controllers/katello/api/v2/repositories_controller.rb#L115